front/app/containers/MainHeader/NotificationMenu/components/InternalCommentNotification/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { memo } from 'react';
+import { isNilOrError, stopPropagation } from 'utils/helperUtils';
+
+// services
+import { IInternalCommentNotificationData } from 'api/notifications/types';
+
+// i18n
+import messages from '../../messages';
+import { FormattedMessage, MessageDescriptor } from 'utils/cl-intl';
+
+// components
+import NotificationWrapper from '../NotificationWrapper';
+import Link from 'utils/cl-router/Link';
+import { DeletedUser } from '../Notification';
+
+interface Props {
+  notification: IInternalCommentNotificationData;
+}
+
+const mapPostTypeToLink = (
+  notification: IInternalCommentNotificationData
+): string => {
+  switch (notification.attributes.post_type) {
+    case 'Idea':
+      return `/admin/projects/${notification.attributes.project_id}/ideas/${notification.attributes.post_id}`;
```

Comment:
Here we are assuming project_id will be string, because the type says it can be null. For ideas, that's probably a correct assumption, but still has a small risk to it.

Reply:
Exactly, it is not possible to be null for an idea, no? Maybe we could add a comment. Any other ideas?

Reply:
you could do if (typeof project_id === 'string') return... and return null by default. But could introduce some extra checks here and there.